### ISSUE TITLE
setLineWidth not working on WebGL

### ISSUE THREAD

**Issue by siauderman** [melonjs/melonJS#994]
Code below. Not working in both 7.1.1 & 8
````
<html>
<head>
	<title>MelonJS setLineWidth not working in WebGL</title>
	<script src="melonjs.js"></script>
	<script>
		let menu=me.Stage.extend({
			init:function(){
				this._super(me.Stage,'init');
			},
			onResetEvent:function(){
				let entity=me.Renderable.extend({
					init:function(){
						let width=me.game.viewport.getWidth();
						let height=me.game.viewport.getHeight();
						this._super(me.Renderable,'init',[0,0,width,height]);
						this.floating=true;
					},
					update:function(dt){
						this._super(me.Renderable,'update',[dt]);
						return true;
					},
					draw:function(renderer){
						renderer.setColor(new me.Color(255,255,0,1));
						renderer.setLineWidth(10);
						let bounds=this.getBounds();
						renderer.strokeLine(bounds.left,bounds.top,bounds.width,bounds.height);
					}
				});
				me.game.world.addChild(new entity());
			}
		});
		me.device.onReady(function(){
			me.video.init(document.body.clientWidth,document.body.clientHeight,{
				wrapper:'screen',
				scale:'auto',
				scaleMethod:'flex',
				renderer:me.video.WEBGL
			});
			me.loader.preload([],function(){
				me.state.set(me.state.MENU,new menu());
				me.state.change(me.state.MENU);
			});
		});
	</script>
	<style>
		#screen{
			position:absolute;
			left:0;
			top:0;
			width:100%;
			height:100%;
		}
	</style>
</head>
<body>
<div id="screen"></div>
</div>
</body>
</html>
````

**Comment by obiot** [comment]
indeed a limitation for now on WebGL, thanks

**Comment by obiot** [comment]
just to be precise on this one, there is some lineWidth implementation on WebGL but that relies on the now obsolete [`lineWidth`](https://developer.mozilla.org/en-US/docs/Web/API/WebGLRenderingContext/lineWidth) method.

**Comment by obiot** [comment]
Closing this one as included in #999
